packages/ironic/charts/values.yaml

```diff
@@ -167,37 +167,18 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 100
-  targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage: 80
-
 tolerations: []
 
 affinity: {}
 
 baremetaloperator:
-  # If no dhpHosts set, all mac addresses acknowledged
-  dhcpHosts: ""
-
-  dnsmasqDnsServerAddress: "{{ .Values.global.dnsmasqDNSServer }}"
-  httpPort: "6180"
```

Comment:
Why did you remove the port when it is used in the Configmap?

Reply:
Elsewhere in the configmap (and deployment) we already hard-code the port, so I just hard-coded the `HTTP_PORT` - so the variable is no longer needed.

Since the port variable wasn't consistently used I don't think we can break existing users with this - it was already hard-coded everywhere except the `HTTP_PORT` value

Reply:
Hmm, Usually when this is smth used on many places, I think the best thing that can be done is to have a Variable in the global Values file and then to use it on all other places. In this way, maintenance will become easier. But this is not related to the PR

Reply:
Yeah I did consider moving it to the global variables - but then realised the current variable doesn't actually work atm, and that it's one of [4 ports in the CM](https://github.com/suse-edge/charts/blob/main/packages/ironic/charts/templates/configmap.yaml#L15..L19) which are all hard-coded

If we decide this needs to be configurable in future we can absolutely re-add a variable, but for now it seemed more consistent to remove the broken variable and align with the other ports which are hard-coded.